package.json

```
@@ -19,7 +19,7 @@
     "stylelint": "stylelint \"source/sass/**/*.scss\" --syntax scss",
     "lint": "npm run editorconfig && npm run stylelint",
     "build": "gulp styles",
-    "start": "gulp"
+    "start": "npx gulp"
```

Если через npm то там npx не нужен обычно
```suggestion
    "start": "gulp"
```

---

Убрала, вроде работает. что так что эдак. Я даже уже не помню зачем они советовали это прописывать))